uc-crux-llvm/src/UCCrux/LLVM/FullType/Type.hs

```diff
@@ -418,7 +472,9 @@ toFullTypeM memType =
               let ?lc = typeContext mts
               Some ftRepr <-
                 case asMemType' strIdent of
-                  Left opaqueIdent -> pure $ Some (FTOpaquePtrRepr opaqueIdent)
+                  Left (L.Ident opaqueIdent) ->
+                    case someSymbol (Text.pack opaqueIdent) of
+                      Some symRep -> pure $ Some (FTOpaquePtrRepr symRep)
```

Comment:
Is the opaque pointer identifier meaningful here? Are they really distinct nominal types, or are there cases where opaque pointers get fudged?

I guess I'm trying to understand what the symbol is representing, what tradeoffs that implies, and if that is actually a safe/desirable thing to represent here.